pandas/core/indexes/range.py

```diff
@@ -428,9 +435,11 @@ def union(self, other):
         -------
         union : Index
         """
```

Comment:
is this needed any longer?

Reply:
see related comment in DatetimeIndex module